### PR TITLE
Add Black River Software to list of consulting companies

### PR DESCRIPTION
**Pull request by eamsden**

### CODE REVIEW

File: commercial-support.tt [hunk 1]
```diff
@@ -12,6 +12,7 @@
 <p>The list is sorted in alphabetical order.</p>
 
 <ul>
+  <li><a href="https://blackriversoft.com/">Black River Software</a></li>
```
Comment (grahamc) [inline commercial-support.tt:15]: It appears the site is down? :grimacing:

Comment (eamsden) [reply commercial-support.tt:15]: Oh, I don't have HTTPS

Comment (eamsden) [reply commercial-support.tt]: Fixed

Comment (SuperSandro2000) [reply commercial-support.tt]: Down for me as well.

Comment (SuperSandro2000) [reply commercial-support.tt]: It shows with curl 🤔

**Pull request merged**